src/crystal/system/win32/hostname.cr

```diff
@@ -0,0 +1,16 @@
+require "c/sysinfoapi"
+
+module Crystal::System
+  def self.hostname
+    retry_wstr_buffer do |buffer, small_buf|
+      name_size = buffer.size.to_u
```

Comment:
Probably `LibC::DWORD.new(buffer.size)` would be better here, in case `to_u` changes.

Reply:
Fixed in 2c59c80.